Dockerfile

```diff
@@ -1,19 +1,34 @@
-FROM alpine:3.3
+FROM alpine:edge
```

Comment:
Weird enough, 3.4 doesn't have packages for PHP 7. So we either stuck with PHP 5 or edge Alpine.